ballerina/Ballerina.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+org = "wso2"
+name = "ballerina_certification_system"
```

Comment:
Let's treat this as the service backend and move the project into service/backend directory instead of ballerina.
Then name accordingly. `ballerina_certification_system` is for entire repo where we can include a web project later